contracts/modules/Exec.sol

```diff
@@ -160,6 +124,7 @@ contract Exec is BaseLogic {
     }
 
     /// @notice Call batchDispatch, but return extra information. Only intended to be used with callStatic.
+    /// @dev The function is deprecated in favor of batchDispatchSimulate with doStaticCall to the lens contract.
```

Comment:
In this function, couldn't we call doBatchDispatch, and then batchDispatch itself could be external instead of public?

Reply:
we're going to change the api of the function (no responses returned anymore), so maybe we should just get rid of it?

Reply:
Yep if it's not too hard to synch the UI update, then let's kill it. I very much doubt there are any other users.